### ISSUE TITLE
Debugger: Improve the error message when vscode attempt to attach to dotnet process with mismatched privilege

### ISSUE THREAD

**Issue by richardzeng1985** [OmniSharp/omnisharp-vscode#477]
## Environment data

`dotnet --info` output:
VS Code version:1.2.1
C# Extension version:1.1.7
## Steps to reproduce
1. Create a sample .dotnet core console application with following code:
   
   ```
   public static void Main(string[] args)
   {
       Console.ReadLine();
       Console.WriteLine("Hello World!");
   }
   ```
2. Launch vscode with **non-admin** and open the project 
3. Launch command line with **admin**, and run the application "dotnet run"
4. Set the target as "attach" and configure the process id of "dotnet" 
5. F5
   ## Expected  behavior

Actually, when I did the testing, I was not aware of launching vscode with non-admin, I suppose this is not supported scenario, however I assume we should at least improve the error message,
1.  At least we should try to remove the internal error message "Unexpected CLRDBG output from command "-target-attach 8508". "  ? 
2. Is it possible to add some useful information similar like "vscode doesn't have the sufficient permission to attach the process" if the permission of vscode and attached process is mismatched ? 
## Actual behavior

Error: No process with the specified id is currently running.

(This is VSDBGE_E_PROCESS_ID_NOT_FOUND)

**Comment by rajkumar42** [comment]
Thanks for reporting. I'm looking into it.

**Comment by WardenGnaw** [comment]
![image](https://cloud.githubusercontent.com/assets/3953714/23573786/8448f91e-002d-11e7-99ac-fa1257c048c0.png)

Internal error message now includes "Access is denied." Should there be another message instead? 

@gregg-miskelly @caslan

**Comment by gregg-miskelly** [comment]
That error message looks reasonable to me. The only thing we might want to do is detect that admin privileges are needed and launch vsdbg as an admin process. But we could also just wait for someone to ask for the feature.

**Comment by kroppt** [comment]
@gregg-miskelly 

This is an issue for me. Is there a way to elevate this process manually? I am kind of a noob when it comes to vscode. What @gregg-miskelly said would be useful.

If you need the manpower, I may be willing to try helping.

**Comment by gregg-miskelly** [comment]
@kroppt What OS are you on?

**Comment by kroppt** [comment]
@gregg-miskelly 
Windows Server 2008 R2 Standard
Service Pack 1
Microsoft's VS Code build

**Comment by gregg-miskelly** [comment]
@kroppt If you close VS Code, right click on it from the start menu/task bar, and choose 'Run as administrator' the debugger should run elevated and therefore be able to attach to the process.

**Comment by kroppt** [comment]
This did not work for me. It is worth noting that dotnet is running under NETWORKSERVICE, which has access to some SQL Active Directory stuff. Administrator should be able to operate on such a process I believe.

I am also kind of a noob when it comes to Windows user permissions.

**Comment by gregg-miskelly** [comment]
@kroppt Yes, I would expect that to work. Can you give me the full error message to see if that gives me any more hints as to what could be going on?

**Comment by kroppt** [comment]
![image](https://user-images.githubusercontent.com/14015672/28466608-97574c3a-6df2-11e7-8931-e204a4a335c0.png)

Access is denied.

**Comment by gregg-miskelly** [comment]
@kroppt I reproduced your issue and I opened up https://github.com/OmniSharp/omnisharp-vscode/issues/1660 to track it. I hope to have a private out later today with the fix.

**Comment by gregg-miskelly** [comment]
On Linux/OSX I think we should consider auto-elevating in this scenario. This is related to https://github.com/OmniSharp/omnisharp-vscode/issues/1811

**Comment by gregg-miskelly** [comment]
Lets see if we can fix this for 1.16, as several people have hit into it now.
